br/pkg/lightning/restore/table_restore.go

```diff
@@ -475,6 +480,9 @@ func (tr *TableRestore) restoreEngine(
 
 	// Restore table data
 	for chunkIndex, chunk := range cp.Chunks {
+		if rc.status != nil && rc.status.backend == config.BackendTiDB {
+			rc.status.FinishedFileSize.Add(chunk.Chunk.Offset)
```

Comment:
should it be `chunk.Chunk.Offset - chunk.Key.Offset` to count previous imported data